microprofile.jdt/org.eclipse.lsp4mp.jdt.test/projects/maven/rest-client-ee9/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have included a project "open-liberty" that should be almost the same as this project (i.e. it uses the `jakarta` namespace). See if you can get away with just using that one (just to save space).